### PR TITLE
(feat): Config to scale the patient identifier sticker content

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR adds a new config point to scale the patient identifier sticker content to fit into the small page sizes.

## Screenshots

https://github.com/user-attachments/assets/a113369b-7fe9-4186-bda2-bd10a7f04487

## Related Issue
None

### CODE REVIEW

File: packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx [hunk 1]
```diff
@@ -46,8 +46,8 @@ const PrintIdentifierSticker: React.FC<PrintIdentifierStickerProps> = ({ closeMo
   const handleAfterPrint = useCallback(() => {
     onBeforeGetContentResolve.current = null;
     setIsPrinting(false);
-    closeModal();
-  }, [closeModal]);
+    // closeModal();
```
Comment (usamaidrsk) [inline packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx:49]: Is this something you forgot re-enable

**Review by usamaidrsk — COMMENTED** [review]
Thanks @vasharma05

File: packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx:49]: Thanks!

File: packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx [hunk 2]
```diff
@@ -64,12 +64,29 @@ const PrintIdentifierSticker: React.FC<PrintIdentifierStickerProps> = ({ closeMo
     setIsPrinting(false);
   }, []);
 
+  const handleInitiatePrint = useCallback(
+    (printWindow: HTMLIFrameElement | null): Promise<void> => {
+      return new Promise<void>((resolve) => {
+        if (printWindow) {
+          const printContent = printWindow.contentDocument || printWindow.contentWindow?.document;
+          if (printContent) {
+            printContent.documentElement.style.setProperty('--print-scale', printScale);
```
Comment (jnsereko) [inline packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx:73]: thank you @vasharma05, don't you think we should automate this? ie We scale basing on the sticker size and available window size.

Comment (jnsereko) [reply packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx:73]: If it brings some level of complex, we can ignore it.

Comment (vasharma05) [reply packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx:73]: I tried automating this, but it didn't work for me. Can you help me in the same?
Thanks!

**Pull request merged**